packages/teleport/src/DesktopSession/useDesktopSession.tsx

```diff
@@ -25,11 +25,11 @@ export default function useDesktopSession(ctx: Ctx) {
   // Tracks combination of tdpclient/websocket and api call state,
   // as well as whether the tdp client for this session was intentionally disconnected.
   const { attempt: fetchAttempt, run } = useAttempt('processing');
-  const { username, desktopId, clusterId } = useParams<UrlDesktopParams>();
+  const { username, desktopName, clusterId } = useParams<UrlDesktopParams>();
```

Comment:
@ibeckermayer  lets keep `desktopId` on the client side, no need to match the backend handler internal implementation. It will be aligned with other ids: `clusterId`, `serverId`. Also keep in mind that this is not actually a `name` but more of `id`.

Reply:
@ibeckermayer after looking at the code it seems that we started using `name` for dbs and kubes. If so, you can keep this change then.

Reply:
although using `name` is really confusing as most of our names are UIDs.